### DISCUSSION TITLE
SOP & Instructions

### DISCUSSION THREAD

**Discussion by mhokama86** [NUWCDIVNPT/stig-manager#684] [Q&A]
Do you have any good SOPs or instructions on setting up?
what about publishing an installer/VM?

**Comment by cd-rite** [comment]
Hi @mhokama86 
We offer several resources, however, proper configuration of any particular deployment will depend on your environment and your organizations security policies. 

Setup documentation, providing guidance on the components required and configuring them here:
[https://stig-manager.readthedocs.io/en/latest/installation-and-setup/index.html](https://stig-manager.readthedocs.io/en/latest/installation-and-setup/index.html)
We offer Docker images, and a sample orchestration:
[https://hub.docker.com/r/nuwcdivnpt/stig-manager](https://hub.docker.com/r/nuwcdivnpt/stig-manager)
We also offer binaries with each release:
[https://github.com/NUWCDIVNPT/stig-manager/releases](https://github.com/NUWCDIVNPT/stig-manager/releases)
